no editor in the container: user edition via command line fails

**How to reproduce:**

1. configure thelounge to start in private mode 
  ```
  module.exports = {
    public: false,
    // ...
  }
  ```  
2. run a container (I run it via docker-compose)
3. add a user (I didn't test this part, I imported a backup)
4. try to edit it (`thelounge_1` is the container name):
  ```
  docker exec -it thelounge_1 lounge edit foobar
  ```

The previous command fails:

> 2016-11-30 22:16:32 [ERROR] Unable to open /home/lounge/data/users/foobar.json. $EDITOR is not set, and vi was not found.

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,6 +5,12 @@ ENV NODE_ENV production
 
 ENV LOUNGE_HOME "/home/lounge/data"
 
+# Install common TTY text editors to allow editing files from within the container
+RUN apt-get update && \
+    apt-get install -y vim nano && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
 RUN mkdir -p "${LOUNGE_HOME}"
 VOLUME "${LOUNGE_HOME}"
 

